flixel/FlxGame.hx

```diff
@@ -259,6 +269,16 @@ class FlxGame extends Sprite
 	}
 	
 	/**
+	 * Sets the filter array to be applied to the game.
+	 * 
+	 * @param	filters
+	 */
+	public function setFilters(filters:Array<BitmapFilter>):Void
```

Comment:
`filters` will cache the object after applying the filters. By default, every time you modify the object the cache will be updated, but because we are using `drawTiles()` on native the object won't be notified about the change so we need to reapply the filters every time we want it to be updated.

Adding this we can control how and when to reapply the filters without the user having to do the same by himself so...